### PR TITLE
LPD-33133 DDMFormField must be multiple if the ObjectField business type is MultiselectPicklist

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://liferay.atlassian.net/browse/LPD-33133

### CODE REVIEW

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/dynamic/data/mapping/storage/test/ObjectDDMStorageAdapterTest.java [hunk 1]
```diff
@@ -0,0 +1,241 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.dynamic.data.mapping.storage.test;
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/dynamic/data/mapping/storage/test/ObjectDDMStorageAdapterTest.java]: Done!

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/dynamic/data/mapping/storage/test/ObjectDDMStorageAdapterTest.java [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+	@After
+	public void tearDown() throws Exception {
+		_objectDefinitionLocalService.deleteObjectDefinition(_objectDefinition);
+
+		_listTypeDefinitionLocalService.deleteListTypeDefinition(
+			_listTypeDefinition);
+	}
+
+	@Test
+	public void testSaveDDMStorageAdapter() throws Exception {
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/dynamic/data/mapping/storage/test/ObjectDDMStorageAdapterTest.java]: @marcelabc I changed to `testSave`

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:object

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:forms

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**